src/App.tsx

```diff
@@ -2,6 +2,32 @@ import './App.css'
 import { useQueryParams } from './hooks/useQueryParams'
 // import reactLogo from './assets/react.svg';
 import { useState } from 'react'
+import React from 'react';
+import { useInterval } from './hooks/use-interval';
+
+const Counter: React.FC = () => {
```

Comment:
@boussadjra , do you think that assigning `React.FC` type is required for the components?